libs/node-client/src/aggregate-instance.ts

```diff
@@ -47,7 +64,73 @@ export class AggregateInstance<
   }
 
   public async process(command: TCommand): Promise<void> {
-    throw new Error('not implemented');
+    const context = this;
+    this.backoff.on('ready', async function () {
+      try {
+        await context.processCommand(command);
+      } catch (err) {
+        if (err instanceof InvalidAggregateVersionError)
+          context.backoff.backoff();
+      }
+    });
+
+    try {
```

Comment:
this is redundant, just call `#backoff()` to execute `#processCommand` immediately

Reply:
the backoffInstance.on read runs on background, calling backoff function wont wait the processCommand to finish inside of backoffInstance.on ready and ends the process function call